src/api/controllers/customers.controller.ts

```diff
@@ -19,6 +25,41 @@ export class CustomersController {
     this.request = new RequestApi(context);
   }
 
+  @logStep('GET /customers with pagination via API')
+  async getCustomersWithPagination(
+    token: string,
```

Comment:
Предлагаю в данном случае сделать один параметр и даже можно попробовать затипизировать его итерфейсом, проще работать с ф-цией будет
```suggestion
    info: { token: string, page?: number, limit?: number, sortField: customersSortField = 'createdOn', sortOrder: sortDirection = 'desc' }
```
Так можно для пейджи и для лимита поставить дефолтное значение и убрать `if`

Reply:
Я бы токен отделял от sortParams, чтобы было getCustomersWithPagination(token: string, sortParams: { page?:number итак далее